### DISCUSSION TITLE
Flow calibration, why is the 2% so much taller?

### DISCUSSION THREAD

**Discussion by limestone-xyz** [supermerill/SuperSlicer#919] [Q&A]
hey all,
does anyone know why the 2% flow calibration is so much taller than the 10% one?
Isn't this just a waste of material?

Cheers,
Gerold

**Comment by supermerill** [comment]
it's a waste of material, but a 2% error is very difficult to see, having more height helps a bit to show over-extrusion.

A even more wasteful and precise version:
print a 50x50x50 cube with 1 perimeter, 1 bottom, no top and 50% rectilinear sparse infill. Then weight it in a precision scale vs the weight given by the slicer (be sure to enter the right filament density)
